source/_posts/django-vs-flask-which-framework-to-choose.md

```diff
@@ -0,0 +1,67 @@
+---
```

Comment:
Link to   previous content -
in content there should be link with previous content

Reply:
Done